lib/diaspora_federation/entities/conversation.rb

```diff
@@ -0,0 +1,12 @@
+module DiasporaFederation
+  module Entities
+    class Conversation < Entity
+      property :guid
+      property :subject
+      property :created_at, default: -> { Time.now.utc }
+      entity :messages, [Entities::Message]
```

Comment:
in which case are multiple messages attached to a conversation for federation? this looks weird.